tutorials/core-concepts/editable-content-on-pages/custom-widgets.md

```diff
@@ -521,6 +521,21 @@ Now is a good time to mention highlights of what you can access in the browser b
 
 And as previously mentioned, you can use LESS in your stylesheets.
 
+### jQuery versions
+
+The default version of jQuery that Apostrophe loads is v1.11.3 for the sake of backward compatibility for older sites. This is and older version that does carry some documented vulnerabilities. For this reason, you can set an option in `apostrophe-assets` to use jQuery v3 instead. This setting is already in place in the `apostrophe-boilerplate` repository for newer projects to use.
+
+If your project isn't already using this, set the `jQuery: 3` option in `apostrophe-assets` in `lib/modules/apostrophe-assets/index.js` or wherever you are configuring assets.
+
+```jquery
+// lib/modules/apostrophe-assets/index.js
+module.exports = {
+  jQuery: 3,
+  // stylesheets, scripts, and other asset configuration
+  // ...
+};
+```
+
 _"What if I want to use browserify, gulp, grunt, etc.?"_ Sure, go nuts. Just arrange your gulpfiles to build a file that is pushed as an asset by one of your modules.
```

Comment:
Just two words and this would cover webpack, and maybe it should only mention webpack now (:

Reply:
Since it has the "etc" I will replace grunt with webpack. I think the community will accept that trade.